src/components/SavedRecipes/SavedRecipes.tsx

```diff
@@ -159,10 +187,10 @@ const SavedRecipes = () => {
           </Flex>
           <Flex flexDirection="column" marginTop="5">
             <Heading as="h3" size="md" marginBottom="3">
-              Tags
+              Tags and Diets
```

Comment:
My personal preference would be to have no heading here since I feel like the buttons are self-explanatory, but you can feel free to ignore me.

Reply:
I will ask team. Thanks

Reply:
I removed it